webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -801,91 +825,92 @@ public static float fitText(Font font, String text, Rectangle rect, float maxFon
         return potentialFontSize;
     }
 
-    private static LinkedList<Chunk> splitScrambleToLineChunks(String paddedScramble, Font scrambleFont, float scrambleColumnWidth) {
+    private static LinkedList<Chunk> splitTextToLineChunks(String text, Font scrambleFont, float scrambleColumnWidth) {
```

Comment:
Since you renamed scramble to text in the method name, shouldn't we also rename all the other occurrences of scramble (`scrambleFont` and `scrambleColumnWidth`)?

Reply:
Fixed